test/cluster/object_store/test_basic.py

```
@@ -869,3 +870,38 @@ async def no_orphaned_objects():
+        # Mark the sstables as "removing" to simulate the problem
+        res = cql.execute("SELECT * FROM system.sstables;")
+        sstables = [row.sstable_id for row in res]
+        logger.debug(f'Found entries: {sstables}')
+
+        scylla_path = await manager.server_get_exe(server.server_id)
+        workdir = await manager.server_get_workdir(server.server_id)
+        args = [scylla_path, "sstable", "dump-scylla-metadata",
+                "--scylla-yaml-file", os.path.join(workdir, "conf", "scylla.yaml"),
+                "--schema-file", schema_file,
+                f"{object_storage.type}://{object_storage.bucket_name}/sstables/{sstables[0]}/TOC.txt"]
+        out = subprocess.check_output(args)
```

Ruff (0.16.1)

[warning] Async functions should not run processes with blocking methods

(ASYNC221)